packages/clerk-js/src/ui/elements/Pagination.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react';
+
+import { Button, Flex, localizationKeys, Text } from '../customizables';
+import { PropsOfComponent } from '../styledSystem';
+import { range } from '../utils';
+
+type UsePaginationProps = {
+  defaultPage?: number;
+};
+
+export const usePagination = (props?: UsePaginationProps) => {
+  const { defaultPage = 1 } = props || {};
+  const [page, setPage] = useState(defaultPage);
+
+  return { page, changePage: setPage };
+};
+
+const PageButton = (props: PropsOfComponent<typeof Button> & { isActive?: boolean }) => {
+  const { sx, isActive, ...rest } = props;
+
+  return (
+    <Button
+      size='xs'
+      variant='ghost'
+      colorScheme='neutral'
+      sx={t => [
+        {
+          color: isActive ? t.colors.$black : t.colors.$blackAlpha700,
+        },
+        sx,
+      ]}
+      {...rest}
+    />
+  );
+};
+
+type RowInfo = {
+  startingRow: number;
+  endingRow: number;
+  allRowsCount: number;
+};
+
+type RowInfoProps = {
+  rowInfo: RowInfo;
+};
+
+const RowInformation = (props: RowInfoProps) => {
+  const {
+    rowInfo: { startingRow, endingRow, allRowsCount },
+  } = props;
+
+  return (
+    <Text>
+      <Text
+        as='span'
+        sx={t => ({
+          color: t.colors.$blackAlpha700,
+        })}
+        localizationKey={localizationKeys('paginationRowText__displaying')}
+      />{' '}
+      <Text as='span'>
+        {startingRow} - {endingRow}
+      </Text>{' '}
+      <Text
+        as='span'
+        sx={t => ({
+          color: t.colors.$blackAlpha700,
+        })}
+        localizationKey={localizationKeys('paginationRowText__of')}
+      />{' '}
+      <Text
+        as='span'
+        sx={t => ({
+          color: t.colors.$blackAlpha700,
+        })}
+      >
+        {allRowsCount}
+      </Text>
+    </Text>
+  );
+};
+
+const shouldShowPageButton = (currentPage: number, pageToShow: number, siblingCount: number, pageCount: number) => {
+  if (Math.abs(currentPage - pageToShow) < siblingCount || pageToShow === pageCount || pageToShow === 1) {
```

Comment:
🙃 This can be simplified as 

```suggestion
  return Math.abs(currentPage - pageToShow) < siblingCount || pageToShow === pageCount || pageToShow === 1;
```

Reply:
✔️